gensim/test/test_ldamodel.py

```diff
@@ -159,57 +159,40 @@ def testEta(self):
             num_topics=2,
             eta=None
         )
-        expected_shape = (2, 1)
+        num_terms = len(dictionary)
+        expected_shape = (num_terms,)
 
         # should not raise anything
         model = self.class_(**kwargs)
         self.assertEqual(model.eta.shape, expected_shape)
-        self.assertTrue(all(model.eta == np.array([[0.5], [0.5]])))
+        self.assertTrue(all(model.eta == np.array([0.5] * num_terms)))
 
         kwargs['eta'] = 'symmetric'
         model = self.class_(**kwargs)
         self.assertEqual(model.eta.shape, expected_shape)
-        self.assertTrue(all(model.eta == np.array([[0.5], [0.5]])))
-
-        kwargs['eta'] = 'asymmetric'
-        model = self.class_(**kwargs)
-        self.assertEqual(model.eta.shape, expected_shape)
-        self.assertTrue(np.allclose(model.eta, [[0.630602], [0.369398]]))
+        self.assertTrue(all(model.eta == np.array([0.5] * num_terms)))
```

Comment:
The "asymmetric" option should not be used for eta. It makes arbitrary words more likely apriori, which does not make sense. I have added an exception in the ldamodel.py file to make sure that `eta` is not equal to "asymmetric".